code_indexer_loop/utils.py

```diff
@@ -1,9 +1,11 @@
 import hashlib
 
 
+# changed "hash_md5" to "md5" to avoid shadowing the built-in "hashlib.md5"
+# rapidmod.io - 04/04/2024
 def hash_md5(filename):
-    hash_md5 = hashlib.md5()
+    md5 = hashlib.md5()
```

Comment:
hash_md5 != hashlib.md5?

Reply:
There is too many shadows, the name makes sense, the inherited un intended consequences do not make sense